client/reader/following/use-subscriptions.ts

```diff
@@ -0,0 +1,40 @@
+import { SubscriptionManager } from '@automattic/data-stores';
+import { useMemo, useEffect } from 'react';
+
+export function useSubscriptions() {
+	const { data: subscriptionsCount, isLoading: isLoadingCount } =
+		SubscriptionManager.useSubscriptionsCountQuery();
+	const {
+		data: siteSubscriptions,
+		isLoading: isLoadingSiteSubscriptions,
+		refetch: refetchSiteSubscriptions,
+	} = SubscriptionManager.useSiteSubscriptionsQuery();
+
+	const isLoading = isLoadingCount || isLoadingSiteSubscriptions;
+	const blogCount = subscriptionsCount?.blogs ?? 0;
```

Comment:
This puts unnecessary optional chaining in dependencies, but is convenient.

Reply:
Nit: `blogsCount`